src/quicktype-core/language/All.ts

```diff
@@ -15,6 +15,7 @@ import { ElmTargetLanguage } from "./Elm";
 import { JSONSchemaTargetLanguage } from "./JSONSchema";
 import { RustTargetLanguage } from "./Rust";
 import { RubyTargetLanguage } from "./ruby";
+import { DartTargetLanguage } from "./dart";
```

Comment:
Oh, I see, it's `Dart`, not `dart`.  It works on macOS because its FS is case-insensitive,.